tests/v2/integration/steveapi/steve_api_test.go

```diff
@@ -1697,6 +2372,11 @@ func (s *steveAPITestSuite) TestList() {
 			}
 			query, err := url.ParseQuery(test.query)
 			require.NoError(s.T(), err)
+			if _, ok := query["sort"]; !ok && s.clusterID != "local" {
+				// k8s does not guarantee any particular order but usually returns results sorted by namespace and name.
+				// k3d seems to have its own ideas, so we can't rely on a consistent order when testing on the downstream cluster.
```

Comment:
😆

Reply:
I did notice the same thing on any downstream cluster :grin: